cockatrice/src/client/ui/widgets/general/display/labeled_input.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by ascor on 10/11/24.
```

Comment:
👀

Reply:
Yes, I'm very proud of this utility class and my continued involvement with the project is contingent on me receiving credit for it.

Just kidding, thanks for the spot :D